cli/README.md

```diff
@@ -43,13 +43,13 @@ keptn [command] [entity] [name] [flags]
 
 where **command**, **entity**, **name**, and **flags** are:
 
-- **command**: Specifies the operation that you want to perform, for example, install, create, onboard, send.
+- **command**: Specifies the operation that you want to perform, for example, install, create, send.
 
-- **entity**: Specifies the entity type. For example, the following commands run a create, onboard, and update operation on the project, service, and domain entity:
+- **entity**: Specifies the entity type. For example, the following commands run a create, get, and update operation on the project, service, and domain entity:
 
     ```console
     keptn create project 
-    keptn onboard service
+    keptn get service

```

Comment:
Can you please change it to `keptn create service`?

Reply:
I've changed it here: 93acf2c0be3d412c6224482a53681c45ba5bb0e3!